hostplumber/main.go

```diff
@@ -51,6 +54,48 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+func validateIPAddressPort(address string) error {
```

Comment:
This function can (and should) be replaced by https://pkg.go.dev/net#SplitHostPort

Reply:
Used this function to get the host and port, and then validated both.